partcad/src/partcad/__init__.py

```
@@ -60,18 +69,20 @@
-    ]
-    debug_to_ignore = [
+        "process_start: ",
+        "process_end: ",
+    )
+    debug_to_ignore = (
         "Starting action",
         "Finished action",
-    ]
+    )
 
     def before_send(event, hint):
```

Ruff (0.8.2)

Missing return type annotation for private function `before_send`

(ANN202)